crd-docs/crd/kiali.io_kialis.yaml

```
@@ -1098,6 +1098,12 @@ spec:
                         description: "Settings used to configure the access url to the Tempo Datasource in Grafana."
                         type: object
                         properties:
+                          cache_capacity:
+                            description: "When 'cache_enabled' is true, the number of traces saved in the cache. 200 by default."
```

use back-ticks so when we publish this on kiali.io, it uses the emphasized font we use for all the other settings
```suggestion
                            description: "When `cache_enabled` is true, the number of traces saved in the cache. 200 by default."
```